### PR TITLE
dracula-icon-theme: init at unstable-2021-07-21

### PR DESCRIPTION
**Pull request by rouven0**
###### Description of changes
Hey there, It's my first contribution in here so I hope it fits all the guidelines^^

This pr adds a gtk icon theme for Dracula.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/icons/dracula-icon-theme/default.nix
Comment (natsukium) [inline pkgs/data/icons/dracula-icon-theme/default.nix:21]: Should we use `jdupes` like other icons?

**Review by natsukium — COMMENTED** [review]
Could you please split the addition of the maintainer list into separate commits?

**Comment by therealr5** [comment]
Thanks for the review. I implemented your suggestions, added jdupes and made the maintainer addition a separate commit.

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 229365` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>dracula-icon-theme</li>
  </ul>
</details>

**Review by natsukium — APPROVED** [review]

File: pkgs/data/icons/dracula-icon-theme/default.nix
Comment (SuperSandro2000) [inline pkgs/data/icons/dracula-icon-theme/default.nix:4]:
```suggestion
```

**Comment by therealr5** [comment]
Any updates on this? There has been radio silence for quite a while now 👀

**Comment by SuperSandro2000** [comment]
> Any updates on this? There has been radio silence for quite a while now 👀

Having a wait time of a week before a PR is merged is quite normal.

File: pkgs/data/icons/dracula-icon-theme/default.nix
Comment (SuperSandro2000) [inline pkgs/data/icons/dracula-icon-theme/default.nix:24]:
```suggestion
    runHook preInstall

```

**Pull request merged**
